book/peer_review.md

```diff
@@ -1,47 +1,78 @@
 # Peer review
 
-Peer review of code is a quality assurance activity, where a developer other than the code's author, views and tests the usage of a piece of code.
+Peer review of code is a quality assurance activity where a developer, other than the code's author, views and tests the usage of a piece of code.
 
 >For analysis to be used to inform a decision it must be possible to assess its utility, reliability, and the degree of validation and verification to which it has been subjected.
 
 [The Aqua book](https://www.gov.uk/government/publications/the-aqua-book-guidance-on-producing-quality-analysis-for-government)
 
-## Why do we need peer review?
+Peer review allows a fresh pair of eyes to take a look at your work. It helps to assure that you have taken an appropriate approach to your analysis and may highlight errors in the analysis process. Constructive review feedback helps you to improve your code quality and provides confidence in your work. It acts to assure that our analysis is fit for purpose.
 
-Peer review allows a fresh pair of eyes to take a look at your work. It validates that you have taken a correct approach to your analysis and may highlight errors. This constructive feedback helps you to improve your code quality and provides confidence in your work. It acts to make sure that our analysis is fit for purpose.
+[The Aqua book](https://www.gov.uk/government/publications/the-aqua-book-guidance-on-producing-quality-analysis-for-government) tells us that quality assurance of our analysis should be proportional to the complexity and business risk of the analysis. This means that both internal and external peer review may be required to adequately assure your analysis. Particularly if your analysis uses novel or complex techniques, where comparison with other analyses cannot be used to challenge your results, or if the analysis is business critical.
```

Comment:
We do mention the responsibilities around Aqua in the Principles section, but it could probably be emphasised. I'm not sure if that's our role?

I've swapped the quotes, as it does make more sense to keep the Aqua one with the related paragraph 👍🏻